### PR TITLE
Node version updates

### PR DESCRIPTION
**Pull request by mattquinlan440**
# Thanks for opening a Pull Request

## Changes

Please outline the reason for the changes you are making:

This change updates the minimum support version of node to the current LTS version (18/hydrogen).

This also removes the usage of the standard-things/esm package as the entry point. Since this package is no longer maintained, newer language features used in some internal dependencies were causing runtime errors. Until we are ready to move this package fully over, we are using babel to transpile down to a cjs friendly version.

Are there any breaking changes you are aware of in the PR?

[(Yes)]

If yes, what are they?

- Updated minimum supported node version
- Changed entry point for the package

## General Checklist

* [x] Change is tested locally
* [x] Demo is updated to exercise change (if applicable)
* [x] [WIP] flag is removed from the title

[1]:https://docs.npmjs.com/about-semantic-versioning
[2]:https://docs.npmjs.com/cli/deprecate
[3]:https://github.com/meltwater/applet-orchard/blob/master/docs/README.md

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,26 +2,24 @@
   "name": "the-orchard",
   "version": "0.2.1",
   "description": "A CLI tool to generate the markup for including dependencies",
-  "main": "src/index.js",
+  "main": "dist/index.js",
   "bin": {
-    "orchard": "src/cli.entry.js"
+    "orchard": "dist/cli.entry.js"
```
Comment (rajeshsusai) [inline package.json]: Not a major concern, just pointing out the `dist/` reference

File: src/index.js
Comment (rajeshsusai) [inline src/index.js:1]: Similar comment to the dist, just pointing out the idea of us using index.js as the entrypoint

File: package.json (hunk 1 above)
Comment (mattquinlan440) [reply package.json]: Good callout. I'll update for consistency

**Review by rajeshsusai — APPROVED** [review]

**Review by jtheriault — APPROVED** [review]
Thank you this LGTM!

**Pull request merged**